flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/DateTimeUtils.java

```diff
@@ -42,7 +42,6 @@
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
-import java.time.temporal.ChronoUnit;
```

Comment:
Update the JavaDoc? Maybe you can copy some of the assumptions that you listed in the PR description as well. In particular, it should explain:
- What belongs here? What doesn't belong here?
- What is the intention of this class? Helper for connectors?

Reply:
Added a comment to clarify the refactoring